### PR TITLE
[PFBF3-0002] Notification Refactor

### LINKED ISSUE #113: [PFBF3-0002] Notification Refactor
### Priority 
- [x] Low
- [ ] Mid
- [ ] High

### Links
Name | Link
-|-
User Story | [link]()
Figma | [link]()

### Completion
Expected | Time
-|-
Expected Start | 19-11-2020
Expected End | 20-11-2020

Success Criteria:
- Rename Notification to HomeNotification for clarity purpose
- Refactor hard coded string
- Remove unused line
- Refactor function and sort it

### PR DESCRIPTION
**Pull request by SetyaG**
Closes #113 
***
## Description
Refactor notification to make it tidier, and do renaming on specified files and functions

## Related Links
List related PRs against other branches:

List | link
------ | ------
User Story | [link]()
Figma | [link]()
Issue | [link](#113 )

## Todos
- [X] UI
- [ ] Coordinator
- [ ] Network
- [ ] Tests
- [X] Documentation

## Screenshot / Video
Screenshot consisting of changes made on this branch.

### UI Changes
UI related changes will be shown here (can be either screenshot or video format)

### Network Status
Network related changes will be shown here (screenshot of request status)

### Test Coverage
Test coverage result for designated changes will be shown here (screenshot of coverage)

## Notes
- No data has been found when using mock API for Sprint 2 (so can't see how it looks when there are data found)

### CODE REVIEW

**Comment by SetyaG** [comment]
HomeNotificationRefactor

File: Laundron-User/LUFeature/LUHome/LUHome/LUHome/HomeNotification/HomeNotificationMessageViewModel+Network.swift [hunk 1]
```diff
@@ -11,16 +11,16 @@ import LUHomeNetwork
 import LUCodable
 import LUHandler
```
Comment (inxudianz) [inline Laundron-User/LUFeature/LUHome/LUHome/LUHome/HomeNotification/HomeNotificationMessageViewModel+Network.swift]: Rename the filename to match class name

File: Laundron-User/LUFeature/LUHome/LUHome/LUHome/HomeNotification/HomeNotificationNewsViewModel+Network.swift [hunk 2]
```diff
@@ -11,14 +11,13 @@ import LUHandler
 import LUCodable
 import LUHomeNetwork
```
Comment (inxudianz) [inline Laundron-User/LUFeature/LUHome/LUHome/LUHome/HomeNotification/HomeNotificationNewsViewModel+Network.swift]: Rename file to match class name

**Review by inxudianz — CHANGES_REQUESTED** [review]

File: Laundron-User/LUFeature/LUHome/LUHome/LUHome/HomeNotification/HomeNotificationNewsViewModel+Network.swift (hunk 2 above)
Comment (SetyaG) [reply Laundron-User/LUFeature/LUHome/LUHome/LUHome/HomeNotification/HomeNotificationNewsViewModel+Network.swift]: moved to a new file name

File: Laundron-User/LUFeature/LUHome/LUHome/LUHome/HomeNotification/HomeNotificationMessageViewModel+Network.swift (hunk 1 above)
Comment (SetyaG) [reply Laundron-User/LUFeature/LUHome/LUHome/LUHome/HomeNotification/HomeNotificationMessageViewModel+Network.swift]: moved to a new file name

**Review by inxudianz — APPROVED** [review]

**Pull request merged**